src/app/core/services/travelperk/travelperk.service.ts

```
@@ -30,7 +30,25 @@
-    return this.apiService.get(`/orgs/${this.orgId}/travelperk/`, {});
+    return this.apiService.get(`/orgs/${this.orgId}/travelperk/`, {}).pipe(
+      catchError(error => {
+        if (error.status === 400 && error.error?.message?.includes('token expired')) {
+          error.error.is_expired = true;
+        }
+        return throwError(() => error);
+      })
+    );
+  }
+
+  getTravelperkTokenHealth(): Observable<{}> {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)